### DISCUSSION TITLE
Publish public as full SPA.

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#1690] [4. General]
Hi
I think livewire is great stack in laravel.
But I think we need one more think which is full SPA or publish public as website.
For example if I put public files in to any site application work fine like SPA which work with API.
Can anyone guide me to do this job with livewire?
@calebporzio please guide me to build this with livewire.

**Comment by iksaku** [comment]
Livewire is not intended to be treated like a SPA, however, you could get a SPA-like feeling if you integrate Turbolinks in your components.
